### PR TITLE
docs: support update and image details updated

### PR DESCRIPTION
**Pull request by rjha-splunk**

### CODE REVIEW

**Review by ikheifets-splunk — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,18 +23,15 @@ For full usage instructions, please visit the Splunk Connect for Syslog [documen
 ## Getting Support
 
 Thank you for considering SC4S for your Splunk needs.
```
Comment (mstopa-splunk) [inline README.md:25]: Do we want to render newlines in this section?

Screenshot 2023-08-09 at 17 06 22

Comment (rjha-splunk) [reply README.md:25]: Yes, i will correct it , thank you

**Review by mstopa-splunk — APPROVED** [review]

**Comment by mstopa-splunk** [comment]
recheck

**Pull request merged**